packages/upload-client/src/index.js

```diff
@@ -144,9 +145,9 @@ async function uploadBlockStream(
           async transform(car, controller) {
             const bytes = new Uint8Array(await car.arrayBuffer())
             // Invoke blob/add and write bytes to write target
-            const multihash = await Blob.add(conf, bytes, options)
+            const { multihash } = await Blob.add(conf, bytes, options)
```

Comment:
let's make sure to create an issue where we can follow up to have user make the site delegation public by default by invoking the Claim delegation

Reply:
created https://github.com/w3s-project/w3up/issues/1486 👍